external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -29,45 +30,36 @@ if(${ENABLE_PCMSolver})
                        -DCMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER}
                        -DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_BINDIR=${CMAKE_INSTALL_BINDIR}
+                       -DPYMOD_INSTALL_LIBDIR=${PYMOD_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_DATADIR=${CMAKE_INSTALL_DATADIR}
                        -DCMAKE_INSTALL_INCLUDEDIR=${CMAKE_INSTALL_INCLUDEDIR}
+                       -DENABLE_psi4=ON
                        -DSTATIC_LIBRARY_ONLY=${_a_only}
+                       -DSHARED_LIBRARY_ONLY=${_so_only}
 #                       -DENABLE_OPENMP=???
                        #-DENABLE_XHOST=???  # option missing
```

Comment:
Yes, this enables one to toggle the a/so of the whole set of external projects at once (and if only one of a/so is built, then that's the one that gets detected). When CheMPS2 was in packaging by one of the Linux distros, it was requested that a shared-only option be added since the distro forbade static libs. So hopefully this is doing PCMSolver a favor in the end.